script.js

```diff
@@ -0,0 +1,88 @@
+$(document).ready(function() {
+
+//builds board
+for (var i = 0; i < 9; i++){
+  appendBox();
+  $(".box").eq(i).attr("id", i);
+  $(".box").eq(i).on("click", function(){
+    game.click++
+    console.log(this.className)
+    console.log(game.click)
+    // this.classList.add("active" + xo())
+    if (this.className === "box") {
+      this.classList.add("active" + xo())
+    } else {
+      game.click--
+      }
+  })
+}
+
+function appendBox() {
+  newDiv = document.createElement('div');
+  newDiv.className = "box";
+  $(".control").append(newDiv)
+}
+
+game = {
+  click: 0,
+  threes: {
+    topRow: [$("#0"), $("#1"), $("#2")],
+    middleRow: [$("#3"), $("#4"), $("#5")],
+    bottomRow: [$("#6"), $("#7"), $("#8")],
+    leftColumn: [$("#0"), $("#3"), $("#6")],
+    middleColumn: [$("#1"), $("#4"), $("#7")],
+    rightColumn: [$("#2"), $("#5"), $("#8")],
+    diagonal1: [$("#0"), $("#4"), $("#8")],
+    diagonal2: [$("#2"), $("#4"), $("#6")]
+  }
+}
+
+function xo(){
+  if (game.click % 2 === 0) {
+    return "X"
+  } else return "O"
+}
+
+$(".control").on("click", function() {
+  threesCompany(game.threes.topRow),
```

Comment:
This is pretty clever! I wonder if there's a way to cut this down a little more..